CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # gulp-static-i18n changelog
 
+## 0.0.2
+
+* Allow sourcing of PO files from Multiple directories
```

Comment:
No. `1.x.x` means that we have a stable release.
We will keep it `0.0.x` to indicate that this code is in flux.
When it gets a bit more mellow we'll bump the minor.
When it's lookin solid, then we'll bump major.